### PR TITLE
[RDS-1281] - Simple token price chart

### PR DESCRIPTION
**Pull request by roszpun**

### CODE REVIEW

File: src/pages/Redstone/Tokens/Tokens.scss [hunk 1]
```diff
@@ -15,4 +15,4 @@
   padding-top: 0.15rem;
   padding-left: 0.3rem;
   padding-right: 0.3rem;
-}
+}
```
Comment (cehali) [inline src/pages/Redstone/Tokens/Tokens.scss:18]: new line

File: src/components/SimplePriceChart/SimplePriceChart.vue [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  content: '';
+  position: absolute;
+  left: 0;
+  top: 0;
+  height: 100%;
+  width: 40px;
+  background: linear-gradient(to right, white, transparent);
+  z-index: 1;
+}
+</style>
```
Comment (cehali) [inline src/components/SimplePriceChart/SimplePriceChart.vue:79]: new line

File: src/mixins/show-more-tokens.js [hunk 3]
```diff
@@ -1,24 +1,31 @@
...
+      this.chartsData[symbol] = valuesMap
+    },
     showMoreTokens() {
       if (this.tokens) {
         let tokensLeftToShow = this.VISIBLE_CHUNK_SIZE;
 
         for (const token of this.tokens) {
           const symbol = token.symbol;
-
+          // this.fetchChartData(symbol, token.providers[0]);
```
Comment (cehali) [inline src/mixins/show-more-tokens.js:28]: leftover?

**Review by cehali — APPROVED** [review]
minor stuff in comments

File: src/mixins/show-more-tokens.js (hunk 3 above)
Comment (roszpun) [reply src/mixins/show-more-tokens.js:28]: This will be later used when I implement it.

**Review by cehali — APPROVED** [review]

**Pull request merged**